### PR TITLE
Improve facebook-js-sdk types

### PR DESCRIPTION
**Pull request by bengry**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:.
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://developers.facebook.com/docs/javascript/reference/v3.3
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/facebook-js-sdk/index.d.ts [hunk 1]
```diff
@@ -1,20 +1,65 @@
...
 declare namespace facebook {
-
+    type FacebookEventType =
+        | 'auth.authResponseChange'
+        | 'auth.logout'
+        | 'auth.login'
+        | 'auth.statusChange'
+        | 'xfbml.render';
+
+    type ResponseStatus =
```
Comment (fluidsonic) [inline types/facebook-js-sdk/index.d.ts]: This is a misleading name.
I'd suggest `LoginStatus` in sync with [FB.getLoginStatus()](https://developers.facebook.com/docs/reference/javascript/FB.getLoginStatus/)

**Review by fluidsonic — CHANGES_REQUESTED** [review]
Looks good except for one type name.

Is there a reason why the whole file has been reformatted with different code style?

File: types/facebook-js-sdk/index.d.ts (hunk 1 above)
Comment (bengry) [reply types/facebook-js-sdk/index.d.ts]: Agreed. Changed.

**Comment by bengry** [comment]
@fluidsonic probably forgot to turn Prettier off before working on the file. I can revert if needed, but I think that at this point Prettier is pretty much the standard in JS formatting, and it makes sense to align with it.

**Comment by amritk** [comment]
@bengry I wouldn't say prettier is the standard, it still has quite a few issues.

**Comment by bengry** [comment]
@amritk sure, some of them are mine even. Nevertheless, it's widely used.

Anyway, if it's an issue I can revert it. Let me know either way. I fixed the naming issue that @fluidsonic raised.

**Comment by amritk** [comment]
I don't mind, as long as the formatting doesn't change every time someone makes a pull request. Up to @fluidsonic. But yea usually when you work on open source files `save without format` ;)

**Review by fluidsonic — APPROVED** [review]

**Pull request merged**